src/raw_statement.rs

```diff
@@ -77,21 +112,23 @@ impl RawStatement {
             }
             rc
         } else {
-            unsafe { ffi::sqlite3_step(self.0) }
+            unsafe { ffi::sqlite3_step(self.ptr) }
         }
     }
 
     pub fn reset(&self) -> c_int {
-        unsafe { ffi::sqlite3_reset(self.0) }
+        unsafe { ffi::sqlite3_reset(self.ptr) }
     }
 
     pub fn bind_parameter_count(&self) -> usize {
-        unsafe { ffi::sqlite3_bind_parameter_count(self.0) as usize }
+        crate::util::get_cached(&self.bind_parameter_count, || unsafe {
+            ffi::sqlite3_bind_parameter_count(self.ptr) as usize
```

Comment:
https://github.com/sqlite/sqlite/blob/master/src/vdbeapi.c#L1584

Reply:
Ah fine. Not worth it then.